container.go

```diff
@@ -860,6 +860,31 @@ func (c *Client) PutContainerArchive(id string, opts PutContainerArchiveOptions)
 	})
 }
 
+// GetContainerArchiveOptions is the set of options that can be used when
+// downloading resources from a container.
+//
+// See https://goo.gl/KnZJDX for more details.
+type GetContainerArchiveOptions struct {
+	OutputStream io.Writer `json:"-" qs:"-"`
+	Path         string    `qs:"path"`
```

Comment:
Yes it is mandatory. I agree its safer, but the're are couple of reasons to leave it inside the opts (and maybe move id back to opts as well):
* The reason i left it inside the opts is to be consistent with code since all other funcs since get opts as params.
* Getting 'path' as param means that we won't use the queryString func formatter and 'path'  must be embedded as part of the url, this fact is also breaking consistency and beauty of code 
* Being repetitive but all funcs has some required params which being transferred as part of the opts struct and it does make since in a way that the definition and the call for those funcs are very clear and readable. Example:  call of func("123", "/file", false) its less clear then calling func(Opts{ID: "123", Path: "/file", Force: false})
Hope i convinced you :).

Can you also give your opinion on my other 'REVIEW' notes please?
Thank you in advance